### PR TITLE
Workflow creation using GitHub Actions (Amazon Linux 2)

### PR DESCRIPTION
**Pull request by sujatadhl**

### CODE REVIEW

File: ansible/roles/nginx/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+- name: Start and enable Nginx
+  systemd:
+    name: nginx
+    enabled: yes
+    state: started
+
+- name: Copy the index.html to web root
+  copy:
+    src: index.html
+    dest: /usr/share/nginx/html/index.html
```
Comment (milandangol) [inline ansible/roles/nginx/tasks/main.yml]: make it variable

File: ansible/roles/nginx/tasks/main.yml [hunk 2]
```diff
@@ -0,0 +1,17 @@
+- name: Install Nginx
+  ansible.builtin.command: sudo amazon-linux-extras install nginx1
+
+- name: Start and enable Nginx
```
Comment (milandangol) [inline ansible/roles/nginx/tasks/main.yml]: use handlers

File: terraform/main.tf [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+    sudo python3.8 -m ensurepip
+    sudo python3.8 -m pip install --upgrade pip
+    sudo yum install git 
+
+    sudo python3.8 -m pip install ansible
+    sudo python3.8 -m pip install botocore
+    sudo python3.8 -m pip install boto3
+
+    ansible-pull -U https://github.com/sujatadhl/pipeline-github.git -i aws_ec2.yml -C github-actions ansible/playbook.yml
```
Comment (milandangol) [inline terraform/main.tf]: As this is public repo. It will work fine. Please think of it as you are using private repo.

**Review by milandangol — CHANGES_REQUESTED** [review]

File: ansible/roles/nginx/tasks/main.yml (hunk 1 above)
Comment (sujatadhl) [reply ansible/roles/nginx/tasks/main.yml]: passed through variable

File: ansible/roles/nginx/tasks/main.yml (hunk 2 above)
Comment (sujatadhl) [reply ansible/roles/nginx/tasks/main.yml]: used handlers

File: terraform/main.tf (hunk 3 above)
Comment (sujatadhl) [reply terraform/main.tf]: changed this command

**Review by milandangol — APPROVED** [review]

**Pull request merged**